controllers/drpolicy_controller.go

```diff
@@ -244,6 +243,18 @@ func (u *drpolicyUpdater) deleteDRPolicy(drclusters *ramen.DRClusterList,
 	ramenConfig *ramen.RamenConfig) error {
 	u.log.Info("delete")
 
+	drpcs := ramen.DRPlacementControlList{}
+	if err := secretsUtil.Client.List(secretsUtil.Ctx, &drpcs); err != nil {
+		return fmt.Errorf("drpcs list: %w", err)
+	}
+
+	for i := range drpcs.Items {
+		drpc1 := &drpcs.Items[i]
+		if u.object.ObjectMeta.Name == drpc1.Spec.DRPolicyRef.Name {
+			return fmt.Errorf("this drpolicy is referenced in existing drpc resource name '%v' ", drpc1.Name)
```

Comment:
Would be useful to add namespace in the message as well. It would be nice also to list all DRPC referencing this policy, but that message may get too long and hence not be appropriate here.